app/containers/App.js

```diff
@@ -95,10 +95,9 @@ const serverdata = {
     UPLOAD_URL: 'https://connect.sensotrend.fi/tpupload',
     DATA_URL: 'https://connect.sensotrend.fi/tpdata',
     BLIP_URL: 'https://www.sensotrend.fi/api'
-  }
+  },
```

Comment:
This file still contains white space changes. See https://github.com/Sensotrend/sensotrend-uploader/pull/58#pullrequestreview-749121679